manifests/kiali-ossm/manifests/kiali.clusterserviceversion.yaml

```diff
@@ -521,26 +521,6 @@ spec:
           - list
           - patch
           - watch
-        - apiGroups: ["authentication.maistra.io"]
-          resources:
-          - servicemeshpolicies
-          verbs:
-          - create
-          - delete
-          - get
-          - list
-          - patch
-          - watch
-        - apiGroups: ["rbac.maistra.io"]
-          resources:
-          - servicemeshrbacconfigs
-          verbs:
-          - create
-          - delete
-          - get
-          - list
-          - patch
-          - watch
```

Comment:
we need to revert this change to the ossm golden copy - in order to support installing the older 1.12 stuff, these permissions are still required.

Reply:
revert PRs:
* https://github.com/kiali/kiali-operator/pull/159
* https://github.com/kiali/kiali-operator/pull/160